components/services/brave_wallet/content/DEPS

```diff
@@ -0,0 +1,4 @@
+include_rules = [
+    "+content/public/browser/service_process_host.h",
```

Comment:
This is typically `+content/public/browser`

Reply:
Done